content/build/cloud/ci.md

```diff
@@ -290,6 +290,37 @@ pipeline {
   }
 }
 ```
+Using the `build.sh` shell script in your `.travis.yml` file under the `install` hook. In this example Travis CI is using Node: 
+
+{{< tabs >}}
+{{< tab name="Travis CI" >}}
+
+```yaml
+language: minimal 
+dist: jammy 
+
+services:
+  - docker
+
+env:
+  global:
+    - IMAGE_NAME=username/repo
+
+before_install: |
+  echo "$DOCKER_PASS" | docker login --username "$DOCKER_USER" --password-stdin
+  sudo apt-get update && sudo apt-get install -y jq build-essential
+  npm --version
+  npm config set loglevel verbose
+  npm config set progress false
+  npm cache clean --force
```

Comment:
There's still a bunch of `npm` commands here, and I'm not sure `build-essential` is needed. The only programs the shell script uses are curl, docker, and jq. Of which curl and docker are already part of the travis base image as far as I understand